### PR TITLE
Prac 10 feedback

### PR DESCRIPTION
**Pull request by AdriantoHartono**
Hi, can you guys help me to check the code?😅
@Lijiayi2
@JevanChandra
@a-rosa
@nora-zyy
@Riston-gitting-gud
@MikeLeongZhiFai

### CODE REVIEW

**Review by MikeLeongZhiFai — COMMENTED** [review]
Code was well written!

**Review by MikeLeongZhiFai — COMMENTED** [review]
I took this as reference! Nice code!

File: prac_10/testing.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    'It is an ex parrot.'
+    >>> changefunction("hello world")
+    'Hello world.'
+    """
+    
+    letter = string[0].upper()
+    note = "".join([letter, string[1:]])
+    if note[-1] != ".":
+        note = "".join([note, "."])
+    return note
```
Comment (Lijiayi2) [inline prac_10/testing.py:92]: well code

File: prac_10/REFLECTION.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Reflections by Adrianto Hartono (134347930)
```
Comment (JevanChandra) [inline prac_10/REFLECTION.md:1]: Good code!

File: prac_10/wiki.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            page_results = wikipedia.page(keyword)
+            print(page_results.title)
+            print(page_results.summary)
+            print(page_results.url)
+        except wikipedia.exceptions.DisambiguationError as e:
+            print(e.options)
+        keyword = input("Please input the search keyword: ")
+    print("Thank you :)")
+
+search()
```
Comment (JevanChandra) [inline prac_10/wiki.py:19]: Nice code

Comment (Lijiayi2) [inline prac_10/wiki.py:19]: all ok, good code

**Pull request merged**